support_policy.md

```diff
@@ -0,0 +1,75 @@
+| 0.12.x        | public.ecr.aws/sagemaker/sagemaker-distribution:0.12-gpu | Sept 1st, 2024 |
+
+## Unsupported Image Versions
+The tables below list SageMaker Distribution Image versions that are no longer supportedand the dates when they reached end of support. AWS will not release any new functionality or security patch updates for these image versions. These unsupported versions will remain visible in the below table for 1 year after their end of support date.
+
+### CPU Images
+
+| Image Version | ECR Image URI | End of Support Date |
+| :---:         | :---:         | :---:               |
+| 1.3.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.3-cpu  | June 28th, 2024  |
+| 1.2.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.2-cpu  | June 28th, 2024  |
+| 1.1.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.1-cpu  | May 2nd, 2024    |
+| 1.0.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.0-cpu  | April 23rd, 2024 |
+| 0.11.x        | public.ecr.aws/sagemaker/sagemaker-distribution:0.11-cpu | July 3rd, 2024   |
+| 0.10.x        | public.ecr.aws/sagemaker/sagemaker-distribution:0.10-cpu | June 6th, 2024   |
+| 0.9.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.9-cpu  | May 3rd, 2024    |
+| 0.8.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.8-cpu  | April 9th, 2024  |
+| 0.7.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.7-cpu  | April 5th, 2024  |
+
+### GPU Images
+| Image Version | ECR Image URI | End of Support Date |
+| :---:         | :---:         | :---:               |
+| 1.3.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.3-gpu  | June 28th, 2024  |
+| 1.2.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.2-gpu  | June 28th, 2024  |
+| 1.1.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.1-gpu  | May 2nd, 2024    |
+| 1.0.x         | public.ecr.aws/sagemaker/sagemaker-distribution:1.0-gpu  | April 23rd, 2024 |
+| 0.11.x        | public.ecr.aws/sagemaker/sagemaker-distribution:0.11-gpu | July 3rd, 2024   |
+| 0.10.x        | public.ecr.aws/sagemaker/sagemaker-distribution:0.10-gpu | June 6th, 2024   |
+| 0.9.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.9-gpu  | May 3rd, 2024    |
+| 0.8.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.8-gpu  | April 9th, 2024  |
+| 0.7.x         | public.ecr.aws/sagemaker/sagemaker-distribution:0.7-gpu  | April 5th, 2024  |
+
+## Frequently Asked Questions
+
+**[Q]** Can I still use older images after an image is no longer supported?
+
+**[A]** Yes, older images remain available in ECR after they reach end of support. However, we highly recommend upgrading to a supported image version that are continuously receiving security updates and bug fixes. It is the customer's responsibility to manage any vulnerabilities that arise due to choosing an image version that is no longer supported by AWS.
```

Comment:
Should we link the shared responsibility model doc? https://aws.amazon.com/compliance/shared-responsibility-model/

Reply:
Yes, I can link this doc.